### PR TITLE
Fix issue link rendering in commit messages

### LINKED ISSUE #2870: Issue indexes in commit messages parser failed.
It should be [Comment help text for issues](#) ([#68](#)), but now it's [Comment help text for issues (#68](#))
image

### PR DESCRIPTION
**Pull request by ethantkoenig**
Fixes #2870.

### CODE REVIEW

File: modules/templates/helper.go [hunk 1]
```diff
@@ -252,28 +253,27 @@ func ReplaceLeft(s, old, new string) string {
 }
 
 // RenderCommitMessage renders commit message with XSS-safe and special links.
-func RenderCommitMessage(full bool, msg, urlPrefix string, metas map[string]string) template.HTML {
+func RenderCommitMessage(msg, urlPrefix string, metas map[string]string) template.HTML {
```
Comment (ethantkoenig) [inline modules/templates/helper.go:256]: I removed the `full` parameter, because it was always set to `false`.

**Comment by lunny** [comment]
image

**Comment by ethantkoenig** [comment]
@lunny fixed

**Comment by lafriks** [comment]
@ethantkoenig is it possible to add so that brackets that are around issue/pr number are not added to commit link?

**Comment by ethantkoenig** [comment]
@lafriks Should be doable

**Comment by lafriks** [comment]
Like in @lunny screenshot in issue. It would look much better if brackets would not be part of any link

**Comment by ethantkoenig** [comment]
@lafriks Done
image

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Nice, LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**